lib/trento/application/usecases/hosts/hosts.ex

```diff
@@ -20,8 +21,18 @@ defmodule Trento.Hosts do
     HostReadModel
     |> where([h], not is_nil(h.hostname) and is_nil(h.deregistered_at))
     |> order_by(asc: :hostname)
+    |> enrich_host_read_model_query()
     |> Repo.all()
-    |> Repo.preload([:sles_subscriptions, :tags, :heartbeat_timestamp])
+    |> Repo.preload([:sles_subscriptions, :tags])
+  end
+
+  @spec get_host_by_id(Ecto.UUID.t()) :: HostReadModel.t()
```

Comment:
Is the signature okey?
What does it return if the id is not found? maybe nil?
just asking